src/calendar/day/period/index.js

```diff
@@ -1,21 +1,18 @@
-import React, {Component} from 'react';
-import PropTypes from 'prop-types';
-import {
-  TouchableWithoutFeedback,
-  Text,
-  View} from 'react-native';
-import {shouldUpdate} from '../../../component-updater';
-import isEqual from 'lodash.isequal';
-
-import * as defaultStyle from '../../../style';
-import styleConstructor from './style';
+import React, { Component } from "react";
```

Comment:
Please remove all formatting changes

Reply:
I agree. Besides, none of these formatting changes are according to `react-native-community`'s ESLint config.